data/presets/presets/amenity/prison.json

```diff
@@ -0,0 +1,21 @@
+{
+    "icon": "prison",
+    "fields": [
+        "operator",
+        "access"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "prison",
+        "cell",
+        "jail"
+    ],
+    "tags": {
+        "amenity": "prison",
+        "barrier": "fence"
```

Comment:
Yeah what @pnorman said - remove `barrier: fence` from here.
People should tag the fence as a separate element, or of they really want to they can add this tag to the amenity using the raw tag editor, but we can't automatically assume that prisons are also fences.